### PR TITLE
Minor comment-only string fixes/uber-nits.

### PR DESCRIPTION
**Pull request by stephendonner**
(Gotta love the GitHub branch name.)

@ncalexan r/merge?  Thx!

### CODE REVIEW

File: docs/examples/android.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 Android Examples
 ================
 
-This page will give a few example to get you up and running with
-browsertime on Android devices.  Generally, add `--android` and then
+This page will give a few examples to get you up and running with
+Browsertime on Android devices.  Generally, add `--android` and then
```
Comment (ncalexan) [inline docs/examples/android.md]: Hmm, I haven't been capitalizing browsertime.  I was writing it as `browsertime`, then dropped the backticks.  Do you capitalize Geckodriver?  I haven't been.  I don't capitalize `ls` (even when I write it ls).

I don't feel strongly about this.

**Review by ncalexan — CHANGES_REQUESTED** [review]
Just to loop back on the capitalization.  Everything else makes sense, thanks.

File: docs/examples/android.md (hunk 1 above)
Comment (stephendonner) [reply docs/examples/android.md]: Thanks for the review, and for the question, @ncalexan!

I would and do capitalize Browsertime (per https://github.com/sitespeedio/browsertime/blob/d7cbe43439cbb4d7b0b44f028387d4a1697dbb0f/README.md and elsewhere), but not geckodriver (per https://github.com/mozilla/geckodriver/blob/79bc0ed40fae4454749530fef9f2a6c5c0e50b59/README.md).

I would and do also capitalize WebDriver (not that you asked, but as an additional example).

I'm fine leaving well enough alone on the s/browsertime/Browsertime changes, if you'd like me to drop those?

Comment (stephendonner) [reply docs/examples/android.md]: (And I should say I strive to leave binary or command-line, etc., names alone, capitalization-wise, for hopefully obvious reasons.)

Comment (stephendonner) [reply docs/examples/android.md]: Fixed; went with prevailing style.

**Comment by stephendonner** [comment]
@ncalexan thanks again for the review; this should be ready to go, with browsertime lower-cased

**Comment by stephendonner** [comment]
@dpalmeiro @brennie (for load-balancing, since I've addressed Nick's comments?)

**Pull request closed without merging**